File: Project/Data/Entities/Address.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

public class Address
{
    [Key]
    public int AddressId { get; set; }

    [Required]
    [StringLength(50)]
    public string? City { get; set; }

    [Required]
    [StringLength(50)]
    public string? Country { get; set; }
}

File: hotel_booking_system_frontend_flutter/lib/Model/address.dart
class Address {
  final int addressId;
  final String city;
  final String country;

  Address({
    required this.addressId,
    required this.city,
    required this.country,
  });

  factory Address.fromJson(Map<String, dynamic> json) {
    return Address(
      addressId: json['addressId'],
      city: json['city'],
      country: json['country'],
    );
  }
}
